### ISSUE TITLE
Untranslatable strings — missing plurals

### ISSUE THREAD

**Issue by piotrdrag** [mozilla/addons-frontend#4609]
“Access your data in %(count)s other domains” in src/amo/components/PermissionsCard/permissions.js:60 and “Access your data on %(count)s other sites” in src/amo/components/PermissionsCard/permissions.js:74 need to be correctly pluralized.

CCing @peiying2

**Comment by bobsilverberg** [comment]
For QA: Just make sure the messages for multiple host permissions for sites and domains are still rendered as expected. You can use https://addons-dev.allizom.org/en-US/firefox/addon/all-of-the-permissions/ as an example.

**Comment by ioanarusiczki** [comment]
Verified this on AMO dev with FF59(Win10 and Android 8.0.0). It looks good to me.
I also checked it for several languages and icons, text appear to be alright.

The windows full screen view

![permissions check](https://user-images.githubusercontent.com/33448286/38132665-a65b5af8-3414-11e8-9de9-aa75a1662390.png)

Android 
![permissions android](https://user-images.githubusercontent.com/33448286/38132870-890bba50-3415-11e8-84c1-77f62a1ee923.png)

**Comment by piotrdrag** [comment]
This hasn’t been fixed. The strings in question are not using ngettext, as you can see at https://github.com/mozilla/addons-frontend/blob/master/locale/pl/LC_MESSAGES/amo.po#L1355

**Comment by tofumatt** [comment]
Shoot, looks like other counts are still untranslated 😢 Re-opening.

**Comment by piotrdrag** [comment]
And I was so happy to see the incorrect statement “Note that we do not need to use ngettext for this as count will never be less than 2.” go. :)

I’m no expert on coding, but it looks like you embedded additional i18n.gettexts inside of i18n.ngettext — compare the MR with e.g. https://github.com/mozilla/addons-frontend/blob/f69eafee2860a60ff4220ca2aa90478d858c3908/src/amo/components/SearchContextCard/index.js#L40

**Comment by tofumatt** [comment]
Argh, shoot. Looks like we messed that up. I reviewed that too, must've just been me being inattentive. Sorry about that 😢

**Comment by piotrdrag** [comment]
No worries! Thanks for quickly looking into this.

**Comment by ioanarusiczki** [comment]
@bobsilverberg Should I re-check plurals for strings ? I'm not sure I understand exactly. 
Thanks!

**Comment by bobsilverberg** [comment]
@ioanarusiczki Yes, it's worth checking the messages for plurals to make sure they are still displayed correctly.

**Comment by ioanarusiczki** [comment]
We have a missing plural - it's the Control browser proxy setting - it is settings not setting. I verified it with the[ support page](https://support.mozilla.org/en-US/kb/permission-request-messages-firefox-extensions)
Should I file this as a follow up? 
Verified with FF59 (Win10).

![control browser proxy setting](https://user-images.githubusercontent.com/33448286/38355577-060a5d18-38c6-11e8-9eb0-011b8fae01ad.png)

**Comment by bobsilverberg** [comment]
@ioanarusiczki Please file a follow-up. This isn't really a pluralization issue, but is an error, as that message should contain an "s" at the end, as per https://searchfox.org/mozilla-central/rev/f860c2bf00576959b07e2d9619c7b5088b0005be/browser/locales/en-US/chrome/browser/browser.properties#117

**Comment by ioanarusiczki** [comment]
Alright, here it is https://github.com/mozilla/addons-frontend/issues/4770
